CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -85,13 +88,20 @@ function Panel3D.Get(panelId)
 
 		panel.transparencyCallbacks = {}
 
+		panel.OnMouseEnter = false
+		panel.OnMouseLeave = false
```

Comment:
We usually set it to nil if you really want to be explicit or nothing at all and you get to save 2 lines of code!